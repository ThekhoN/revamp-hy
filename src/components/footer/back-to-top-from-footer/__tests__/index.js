import React from 'react';
import BackToTopFromFooter from '../index';
import renderer from 'react-test-renderer';

describe('BackToTopFromFooter', () => {
	it('renders correct snapshot', () => {
		const tree = renderer.create(<BackToTopFromFooter />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
