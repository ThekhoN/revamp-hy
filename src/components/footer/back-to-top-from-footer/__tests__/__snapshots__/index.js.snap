// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackToTopFromFooter renders correct snapshot 1`] = `
<div
  className="back-to-top-from-footer"
>
  Back to top
</div>
`;
